x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/stats/FeatureMetric.java

```diff
@@ -49,7 +48,7 @@ public enum FeatureMetric {
     DROP(Drop.class::isInstance),
     KEEP(Keep.class::isInstance),
     RENAME(Rename.class::isInstance),
-    META(MetaFunctions.class::isInstance);
+    META(p -> false);
```

Comment:
Let's add a comment why this exists and why it shouldn't be removed.

Actually, can this be removed? I think @astefan or @luigidellaquila know more about feature metrics and could know if old metrics need to be kept around.

Reply:
Let me dig more. I had thought removing it would be dangerous for serialization of the metric. I'll look!

Reply:
Looking more, I'm not sure why we have the note about the ordering here. It *looks* like no instances of this class escape a node. We use strings for serializing it. So maybe we're safe here. Can I just delete this?

Reply:
Metrics are not serialized between nodes, and the Stats API relies on the metric name, not on the enum ordinal (that is only used during the metric collection phase, to populate a bitset).
I _think_ it's safe to remove, but please wait for @astefan